src/patternfly/components/Select/select.scss

```diff
@@ -23,8 +21,6 @@
   --pf-c-select__toggle--expanded--BorderBottomWidth: var(--pf-global--BorderWidth--md);
   --pf-c-select__toggle--expanded--BorderBottomColor: var(--pf-global--active-color--100);
   --pf-c-select__toggle--m-plain--BorderColor: transparent;
-  --pf-c-select__toggle--m-plain--Color: var(--pf-global--Color--200);
-  --pf-c-select__toggle--m-plain--hover--Color: var(--pf-global--Color--100);
```

Comment:
Are these two lines breaking changes

Reply:
Hmm... technically yes, but it was never a version of the component that we supported. It wouldn't make sense to have the "plain" code we have here for the select. I'm fine with deleting it but we could be safe and remove it later. What do you think?

Reply:
I think be safer and remove it later. Just in case someone is using one of these variables, even though we aren't using them